docs/config-file/v2.rst

```diff
@@ -284,6 +284,7 @@ The builder type for the Sphinx documentation.
 
 :Type: ``string``
 :Options: ``html``, ``htmldir``, ``singlehtml``
+:Options: ``html``, ``dirhtml``, ``singlehtml``
```

Comment:
You should modify the line above, not add a new one. Changing `htmldir` for `dirhtml` (here is fine to replace the old value)

Reply:
Maybe adding a note about the old value was deprecated, so users don't get confused.

Something like: The `htmldir` builder option was renamed to `dirhtml` to use the same name as sphinx. Configurations using the old name will continue working.